package.json

```diff
@@ -10,9 +10,10 @@
     "url": "https://github.com/erikras/redux-form"
   },
   "scripts": {
+    "analyze": "webpack src/index.js dist/redux-form.js -p --bail --profile --json > stats.json && webpack-bundle-analyzer stats.json",
```

Comment:
The current versions of these tools seem to require having an actual `stats.json` on the filesystem.